docs/DeveloperGuide.md

```diff
@@ -542,6 +542,25 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 _Similar to UC10 except without extension 2b._
 
+**Use case: UC12 - Undo a command**
+
+**MSS**
+
+1.  Actor performs a command that updates the addressbook.
+2.  System executes the command.
+3.  Actor requests to undo the recently executed command.
+4.  System reverts changes made by the user.
```

Comment:
Should user be Actor here?

Reply:
I think it is fine since actor we have defined it at the start of the DG